client-app/shared/catalog/components/product-price-block.vue

```diff
@@ -30,23 +30,45 @@
         </span>
       </div>
 
-      <div
-        class="flex items-center justify-center flex-1 select-none py-4 px-1 border-r space-x-2 cursor-pointer hover:bg-gray-100"
-      >
-        <i class="fas fa-envelope fa-xl text-[color:var(--color-primary)]" />
-        <span class="text-sm text-blue-800 font-bold">
-          {{ $t("shared.catalog.product_details.price_block.send_email_button") }}
-        </span>
-      </div>
+      <div class="flex w-3/5">
+        <div class="w-1/3">
+          <VcPopover :title="$t('shared.catalog.product_details.share_product_label')" :showCloseButton="true">
+            <template #trigger>
+              <div
+                class="items-center justify-center select-none py-4 px-1 border-r space-x-2 cursor-pointer hover:bg-gray-100"
+              >
+                <i class="fas fa-share-square fa-xl text-[color:var(--color-primary)]" />
+              </div>
+            </template>
+            <template #content>
+              <div class="flex justify-between items-center text-xl px-5 mb-3 h-16">
+                <a
+                  class="w-14"
+                  :href="stringFormat(socialSharingService.url_template, pageUrl)"
```

Comment:
Maybe it be more readeble if move it a small code to method with name some like getSocialPostUrlForProduct()